### ISSUE TITLE
What is needed for implementing Bikekeep in similar manner as Bikely

### ISSUE THREAD

**Issue by tsobuskerudbyen** [buskerudbyen/cycling-norway#102]
Please have a look: https://docs.bikeep.com/#overview. How much work is needed here? Can this then be done in OTP as well?

**Comment by leonardehrenfried** [comment]
Is Bikeep is a service for locking and unlocking parking facilities? If so then it's very much outside of what we (as the maintainers) see as in-scope for OTP.

What we could do is get the locations from their API and display them on a map or do routing with them in OTP.

I have colleagues in Germany that are maintaining an open source stack for these bike lockers. If you want I can put you in touch.

**Comment by tsobuskerudbyen** [comment]
Norwegian translations:
-  `BIKE_HOUSE` = "Bikeep sykkelparkeringshus" (Not the same app as Sykkelhotell, hence we name them differently)
-  `PRIVATE` = "Bikeep sykkelstasjon" - We should, per now, reuse the normal cycle parking symbol here then. And add number of available private stations of course.

Icons used in the app for reference:
![Screenshot_20240322_134951_Bikeep](https://github.com/buskerudbyen/cycling-norway/assets/70061303/cb5eb750-908c-4ef0-bf89-d2ebbb020c0a)

**Comment by tsobuskerudbyen** [comment]
@leonardehrenfried Back from Easter holidays. Can you please link OTP ticket with this ticket?

**Comment by tsobuskerudbyen** [comment]
OTP: https://github.com/opentripplanner/OpenTripPlanner/pull/5798 - "Add Bikeep bicycle parking updater"

**Comment by Beck-berry** [comment]
> Norwegian translations:
> * `BIKE_HOUSE` = "Bikeep sykkelparkeringshus" (Not the same app as Sykkelhotell, hence we name them differently)
> * `PRIVATE` = "Bikeep sykkelstasjon" - We should, per now, reuse the normal cycle parking symbol here then. And add number of available private stations of course. One location for a single bike is called: "Bikeep sykkelstativ"

The tags that are used (and I found them so far): "PRIVATE", "FREE", "BOOKABLE", "BIKE_HOUSE", "BIKE".
There are locations that can be `BIKE_HOUSE` and `PRIVATE` at the same time, so I would advice not to change icons and legend based on this, only add the tag data as a description at the popup.
What do you think, @tsobuskerudbyen? Do you know, what all of these mean? If it's ok for you, we'd need descriptions for all the tags.

**Comment by tsobuskerudbyen** [comment]
Yeah, I see the confusion here. As far as I know the `BIKE_HOUSE` takes precedence, as it contains x numbers of `PRIVATE` stations inside. Icon on the first: Are there any "house with a bike in it"? Something like this: 
![image](https://github.com/buskerudbyen/cycling-norway/assets/70061303/ad03a7cb-4665-4bde-9967-01ecf5eeb7ea)
Copied from: <a href="https://www.flaticon.com/free-icons/cycle" title="cycle icons"> Cycle icons created by manshagraphics - Flaticon</a> 

Also, see the light green icon from the app screenshot over. Looks a bit strange that is has a person on the bike.

**Comment by Beck-berry** [comment]
As I mentioned earlier, we can create a switch in the legend for _separate layers only_. Both bikehouses and private parkings will have the same switch: bikeep parkering. Since the switch works for both type of bike parkings (bike house and private), I would suggest that we do not use different icons for them, because it could confuse users; only show the difference in the popup. It is understandable of Bikeep to make a difference between its own services, but we display many different layers already.
Anyways, I will check how the legend/switch handles if there are multiple icons for a layer and come back to this.

----

Added different icons for bikeep:
at the legend:
![Image](https://github.com/buskerudbyen/cycling-norway/assets/46535522/77dafb51-2260-4efd-b996-66df562bec14)
at the map:
![Image](https://github.com/buskerudbyen/cycling-norway/assets/46535522/7d172a61-1cc3-4eac-bd29-49634a6d0abd)

Added its own popup:
if a private parking:
![Image](https://github.com/buskerudbyen/cycling-norway/assets/46535522/4a2d1f04-1eb1-4a27-b28d-bfa46f3b2be3)
if a bikehouse:
![Image](https://github.com/buskerudbyen/cycling-norway/assets/46535522/f356f9e8-09f3-488c-a770-106c5a2db3d6)
